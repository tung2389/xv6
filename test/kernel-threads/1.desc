Test shared memory, lock and create threads.