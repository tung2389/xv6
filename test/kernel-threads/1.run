cd src; ../tester/run-xv6-command.exp CPUS=1 Makefile.test thread_test_1 | grep XV6_TEST_OUTPUT; cd ..