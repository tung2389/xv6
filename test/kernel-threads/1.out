XV6_TEST_OUTPUT: process's shared_global value: 99
XV6_TEST_OUTPUT: process's heap_var value: 9999
XV6_TEST_OUTPUT: thread's shared_global value: 99
XV6_TEST_OUTPUT: thread's heap_var value: 99
XV6_TEST_OUTPUT: thread's shared_global value: 99
XV6_TEST_OUTPUT: thread's heap_var value: 99
